### PR TITLE
add support for timescaleDB according to issue #1668

### PR DESCRIPTION
**Pull request by Ashutosh619-sudo**
added support for timescaleDB in mindsdb data source according to issue #1668.

### CODE REVIEW

File: mindsdb_datasources/datasources/timescale_ds.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import os
```
Comment (StpMax) [inline mindsdb_datasources/datasources/timescale_ds.py]: unnecessary import

**Review by StpMax — CHANGES_REQUESTED** [review]
also need add import of this class to `__init__.py`

File: mindsdb_datasources/datasources/timescale_ds.py (hunk 1 above)
Comment (Ashutosh619-sudo) [reply mindsdb_datasources/datasources/timescale_ds.py]: On it.

Comment (Ashutosh619-sudo) [reply mindsdb_datasources/datasources/timescale_ds.py]: done all the changes!

**Review by ZoranPandovski — APPROVED** [review]
Thanks

**Pull request merged**